src/connector/Cargo.toml

```diff
@@ -70,6 +70,7 @@ rdkafka = { workspace = true, features = [
 ] }
 reqwest = { version = "0.11", features = ["json"] }
 risingwave_common = { path = "../common" }
+risingwave_expr = { path = "../expr" }
```

Comment:
We shall avoid this dependency by moving `str_to_bytea` to the `risingwave_common` crate. #9629

Reply:
I see, let me move it.